pages/home/quickstart.mdx

```diff
@@ -69,6 +68,8 @@ touch example_openai.js
 </Tabs.Tab>
 </Tabs>
 
+Next we'll edit that file and add some code. Open the `example_openai.js` file you just created in your preferred editor. 
```

Comment:
this says to open the `js` file even in the python tab.

Reply:
I think this was inserted in a previous comment. I just removed the file name, there's no need for it. We already reference as "the file you just created"